src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -309,7 +303,6 @@ protected override void OnScriptFileChanged(object sender, FileSystemEventArgs e
                 if (rawBody != null)
                 {
                     requestObject["rawBody"] = rawBody;
-                    bindDataInput = rawBody;
```

Comment:
This subtle change means binding data for Node http scenarios is now coming from the new HttpTrigger binding, since ApplyBindingData below only extracts binding data if its input is a json string.